state/redis/redis.go

```diff
@@ -318,6 +346,19 @@ func (r *StateStore) parseETag(req *state.SetRequest) (int, error) {
 	return ver, nil
 }
 
+func (r *StateStore) parseTTL(req *state.SetRequest) (int, error) {
```

Comment:
Please, send a new PR later centralizing the parseTTL code since it is duplicated in many state stores.

Reply:
Sure!